src/app/components/header/header.component.html

```diff
@@ -4,11 +4,21 @@
   <div
     class="user-container h-full w-full flex justify-start items-center gap-2"
   >
-    <p
-      class="bg-seasalt w-fit h-fit rounded-3xl px-3 py-1.5 text-xl font-pacifico text-pale-brown"
-    >
-      Y
-    </p>
-    <p class="font-tilt-neon font-semibold text-lg text-pale-brown">Hi Yané</p>
+    @if (loggedInUser !== null && loggedInUser !== "" && loggedInUser !== undefined) {
```

Comment:
this is still from when we were testing, but you can change this back to `loggedInUser ? loggedInUser : ''`, the whole if statement is a bit verbose, but not wrong